ansible/Vagrantfile

```diff
@@ -0,0 +1,106 @@
+# -*- mode: ruby -*-
```

Comment:
This file needs to exist in the root. I think the only change is to `ANSIBLE_PATH`.